project/ProjectPlugin.scala

```diff
@@ -65,7 +65,9 @@ object ProjectPlugin extends AutoPlugin {
 
   override def projectSettings =
     Seq(
-      testFrameworks in Test := Seq(TestFrameworks.Specs2),
+      testFrameworks in Test := Seq(TestFrameworks.Specs2,
```

Comment:
:clap: :clap: you didn't even have to ask! How long did this take you to figure out? :rofl:

Reply:
You need to start writing that sbt book faster before this drives me insane